.circleci/config.yml

```diff
@@ -123,6 +123,9 @@ commands:
 
             drupal_release_name=`silta ci release name`
             next_release_name=`silta ci release name --release-suffix next`
+            # This is the name that will be used in the internal URL, used in
+            # Node containers to call the Drupal backend.
+            drupal_release_name_for_internal_use=${drupal_release_name}-drupal
```

Comment:
I'm wondering whether this works with long branchnames that get truncated and hashed. `silta ci release name --release-suffix drupal` might work better? One way to test this would be to just create a long branchname and push it to see if it works after deployment.

Side note: Not sure if the manual truncation in this script is still needed as `silta ci release name` will already truncate long names, including the release suffixes.

Reply:
It looks like it would work, but it's incorrect usage. The reason - you use `release-suffix: next` and `release-suffix: storybook` but for drupal there is no release suffix, just release name. And `-drupal` is a "load balancer hostname" (service)

Reply:
Thanks for clarifying this!